tools/testsys/src/vmware_k8s.rs

```diff
@@ -0,0 +1,241 @@
+use crate::aws_resources::migration_crd;
+use crate::crds::{
+    BottlerocketInput, ClusterInput, CrdCreator, CrdInput, CreateCrdOutput, MigrationInput,
+    TestInput,
+};
+use crate::error::{self, Result};
+use crate::sonobuoy::sonobuoy_crd;
+use bottlerocket_types::agent_config::{
+    CreationPolicy, K8sVersion, VSphereK8sClusterConfig, VSphereK8sClusterInfo, VSphereVmConfig,
+};
+use maplit::btreemap;
+use model::{Crd, DestructionPolicy};
+use pubsys_config::vmware::Datacenter;
+use snafu::OptionExt;
+use std::collections::BTreeMap;
+use std::str::FromStr;
+
+/// A `CrdCreator` responsible for creating crd related to `aws-k8s` variants.
+pub(crate) struct VmwareK8sCreator {
+    pub(crate) region: String,
+    pub(crate) datacenter: Datacenter,
+    pub(crate) ova_name: String,
+    pub(crate) encoded_mgmt_cluster_kubeconfig: String,
+}
+
+#[async_trait::async_trait]
+impl CrdCreator for VmwareK8sCreator {
+    /// Use the `BUILDSYS_OVA` name for the image id.
```

Comment:
Is `BUILDSYS_OVA` name the only place that this can come from? It strikes me as odd to see this comment far away from the place where the value has been set. This function is just a getter.